R/step_naomit2.R

```diff
@@ -0,0 +1,13 @@
+#' Special NA omit step that does two steps in one
+#'
+#' @param x Recipe to be used for omission steps
+#'
+#' @return Omits NA's from both predictors and outcomes and training time;
+#' however, only omits predictors at prediction time
+#' @export
+
+step_naomit2 <- function(x) {
```

Comment:
I'm not sure if it's common to add a number in a function name? Perhaps something like step_naomit_all()?

Reply:
Yeah, this really needs a better name. I'm having less and less fondness for prefixing everything with `epi_`, but for now, maybe `step_epi_naomit()`

Reply:
Fixed this. I just got the idea of the "2" from how there also exists `read_csv2(...)`.